feed/tests/test_comment_model.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+from feed.models import Post, Comment
+import django.contrib.auth
+
+
+User = django.contrib.auth.get_user_model()
+POST_TITLE = "Some First Post Title"
+POST_CONTENT = "Some first post content here"
+
+
+@pytest.fixture
+def users(db):
+    user0 = User.objects.create_user('user1', password='pass123')
+    user1 = User.objects.create_user('user2', password='pass321')
+    return [user0, user1]
+
+
+@pytest.fixture
+def post0(db, users):
+    return Post.posts.create(title=POST_TITLE, content=POST_CONTENT, author=users[0])
+
+
+COMMENT1_CONTENT = "Some first comment content goes here"
+COMMENT2_CONTENT = "Some second comment content goes here"
+
+
+@pytest.fixture
+def comments(post0, users):
+    comment0 = Comment(author=users[1], post_parent=post0, content=COMMENT1_CONTENT)
+    comment1 = Comment(author=users[0], post_parent=post0, comment_parent=comment0, content=COMMENT2_CONTENT)
+    return [comment0, comment1]
+
+
+@pytest.mark.django_db
+class TestCommentModel:
+    # Testing different comment attributes
+
+    def test_new_comment_properties(self, comments, users, post0):
+        # Testing comment data
+        assert comments[0].author == users[1]
+        assert comments[0].post_parent == post0
+        assert comments[0].content == COMMENT1_CONTENT
+        assert comments[1].comment_parent == comments[0]
+
+    def test_comment_save_to_db(self, comments):
+        # Testing comment save in DB
+        comments[0].save()
+        assert comments[0] in Comment.comments.all()
```

Comment:
same here, the tests sgould filter the comment in db, and compare it against the returned data from fixtures

Please apply for all the rest of the tests

Reply:
Yes, but please compare it with all comments data

Reply:
Im not sure what do you mean by compare to all comments data, do you mean to filter according to all properties and not only to content?

Reply:
This is the purpose of the test, 
Ig you have a comment, how do you find the logical connections from db, FYI